DESCRIPTION.md

```diff
@@ -14,6 +14,9 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
   - Fixed issue with connection diagnostics failing to complete certificate checks.
   - Fixed issue that arrow iterator causes `ImportError` when the c extensions are not compiled.
   - Removed dependencies on Cryptodome and oscrypto and removed the `use_openssl_only` parameter. All connections now go through OpenSSL via the cryptography library, which was already a dependency.
+  - Added the `backoff` argument to `snowflake.connector.connect` allowing for configurable backoff policy between retries of failed requests. See available implementations in the `time_util` module.
+  - Added the `socket_timeout` argument to `snowflake.connector.connect` specifying socket read and connect timeout.
+  - Fixed `login_timeout` and `network_timeout` (renamed to `request_timeout` for clarity) behaviour. Retries of login and network requests are now properly halted after these timeouts expire.
```

Comment:
This will need docs change https://docs.snowflake.com/en/developer-guide/python-connector/python-connector-api#functions

Reply:
You mentioned there was someone I should talk to for docs changes.

Reply:
Just pinged you about this privately